ios/WebRTC/WebRTC.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name             = "WebRTC"
+  s.version          = "0.0.112"
+  s.summary          = "WebRTC dependency"
+  s.homepage         = "https://github.com/CoSMoSoftware/libwebrtc"
+  s.license          = { :type => "MIT", :text => "MIT License" }
+  s.author           = { "Aravind Raveendran" => "aravind.raveendran@dolby.com" }
+  s.source           = { :http => 'file:' + __dir__ + '/../libWebRTC.zip' }
+  s.ios.deployment_target   = "12.4"
+  s.tvos.deployment_target  = "12.4"
+  s.vendored_frameworks     = [ 'libWebRTC/Frameworks/WebRTC.xcframework' ]
+  s.source_files            = "libWebRTC/Frameworks/*.{h,m,swift}"
+  s.preserve_paths          = "*"
+end
```

Comment:
Getting an error, is some dependency is missing?

/Users/ldani/StudioProjects/rts/streaming-sdk-react-native-getting-started/node_modules/react-native-webrtc/ios/RCTWebRTC/WebRTCModuleOptions.h:2:9: 'WebRTC/WebRTC.h' file not found

/Users/ldani/StudioProjects/rts/streaming-sdk-react-native-getting-started/node_modules/react-native-webrtc/ios/RCTWebRTC/WebRTCModuleOptions.m:1:9: in file included from /Users/ldani/StudioProjects/rts/streaming-sdk-react-native-getting-started/node_modules/react-native-webrtc/ios/RCTWebRTC/WebRTCModuleOptions.m:1:

Reply:
Needs a fix in https://github.com/montanaeli/react-native-webrtc repo for a successful build. That's pending.